pkg/storage/tsdb/testutil/block_generator.go

```diff
@@ -63,11 +63,6 @@ func GenerateBlockFromSpec(_ string, storageDir string, specs BlockSeriesSpecs)
 	blockID := ulid.MustNew(ulid.Now(), rand.Reader)
 	blockDir := filepath.Join(storageDir, blockID.String())
 
-	// Ensure series labels are sorted.
-	for _, series := range specs {
-		sort.Sort(series.Labels)
-	}
```

Comment:
I don't have enough context where this is called from, but I guess you know what you're doing.